linkerd/tracing/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![deny(warnings, rust_2018_idioms, clippy::disallowed_method)]
+#![deny(warnings, rust_2018_idioms)]
```

Comment:
reverted?
```suggestion
#![deny(warnings, rust_2018_idioms, clippy::disallowed_metho)]
```

Reply:
agh, that was also a merge mistake --- thanks for catching that